src/openMVG/sfm/sfm_data_BA_ceres.cpp

```diff
@@ -180,27 +183,117 @@ bool Bundle_Adjustment_Ceres::Adjust(
     }
   }
 
+  Hash_Map<IndexT, std::size_t> intrinsicsUsage;
+
+  // Setup Intrinsics data & subparametrization
+  const bool refineIntrinsicsOpticalCenter = (refineOptions & BA_REFINE_INTRINSICS_OPTICALCENTER_ALWAYS) || (refineOptions & BA_REFINE_INTRINSICS_OPTICALCENTER_ALWAYS);
```

Comment:
it the same condition in OR, maybe you meant `BA_REFINE_INTRINSICS_OPTICALCENTER_IF_ENOUGH_DATA` for the second?